tests/unit/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  env: {
+    jest: true
+  }
+}
```

Comment:
should this not be the same .eslintrc as in the outer project? why have two?

Reply:
There might be some rules that might need to be different so the linter doesn't fail on the test code (for example, the linter *might* freak out since we're using keywords like `describe` and `beforeEach` that aren't imported like in other code because the test runner defines these global functions.

We could try to copy the existing `ESLint` configuration here and see if the build freaks out, and then modify the rules accordingly until it passes.

Reply:
Copied configuration but changed no-undef to a warning